example/eblob_backend.c

```diff
@@ -976,7 +988,9 @@ static void eblob_backend_cleanup(void *priv)
 {
 	struct eblob_backend_config *c = priv;
 
-	eblob_cleanup(c->eblob);
+	if (c->eblob) {
```

Comment:
The only place where we initialize locks and data is dnet_blob_config_init(), this is also the only place where we initialize $eblob field. How is it possible to call cleanup() with half initialized backend config structure?

Also, it looks like cleanup can not be called without proper backend config initialization at all. Since it is exported to c++ world only via init/cleanup callbacks.